db_stress_tool/db_stress_common.h

```diff
@@ -150,6 +150,7 @@ DECLARE_bool(use_block_based_filter);
 DECLARE_int32(ribbon_starting_level);
 DECLARE_bool(partition_filters);
 DECLARE_bool(optimize_filters_for_memory);
+DECLARE_bool(detect_filter_construct_corruption);
```

Comment:
Question: 
I feel like it does not hard to add this to stress test since we can test it with other random options. It is just that I don't know what to expect from stress test IF this feature did detect a filter construction corruption. For example, I manually injected some construction error and it gives me terminate and core dump when running `./db_stress --detect_filter_construct_corruption=true` - is it what we want from stress test?

Reply:
Hmmm .. just to make sure I understand for my future reference: suppose there exists a filter construction corruption and detect_filter_construct_corruption=true catches it in db_stress, which now will lead to db_stress run terminates and core dumps. Will this test run be classified as "crash test failure" (like the one we need to investigate into)? If we do, do we need to change the way of handling such corruption happening in db_stress?